src/components/Card/Card.js

```diff
@@ -1,12 +1,23 @@
+import { useState } from 'react';
+import { Button } from '../UI/Button.styled';
 import { CardContainer } from '../UI/CardContainer.styled';
 import Typography from '../UI/Typography';
 
-export default function Card({ category, name, address }) {
+export default function Card({ category, name, address, products, information, visited, rating }) {
+	const [show, setShow] = useState(false);
+	const handleOpen = () => {
+		setShow(!show);
+	};
 	return (
-		<CardContainer>
+		<CardContainer onClick={handleOpen}>
 			<Typography variant="p">{category}</Typography>
 			<Typography variant="h2">{name}</Typography>
 			<Typography variant="p">{address}</Typography>
+			<Button>{show ? 'Weniger anzeigen' : 'Mehr anzeigen'}</Button>
+			{show && <Typography variant="p">{products}</Typography>}
+			{show && <Typography variant="p">{information}</Typography>}
+			{show && <Typography variant="p">{visited}</Typography>}
+			{show && <Typography variant="p">{rating}</Typography>}
```

Comment:
Think about separating this four Typography Components into one seperate component, so you only need to 
`{show && <TheNewComponentYouWrote/>}` once to have the same effect.

Reply:
We already talked about this, I will approve your PR but maybe try rendering this information within a list in future.

Reply:
Thanks! :)